examples/pytorch/Logging/pytorch log model.ipynb

```diff
@@ -0,0 +1,416 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "7066087c-6947-4a50-a7fb-6ab71c004ccf",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Log a pytorch model\n",
+    "Note: you can also use this workflow for HuggingFace models, as they're torch models under the hood!\n",
+    "\n",
+    "In this guide we will:\n",
+    "- define a simple pytorch model\n",
+    "- set a model signature for our logged model to define inputs and outputs to the mlflow model\n",
+    "- create an mlflow experiment, where we will log runs and their corresponding models\n",
+    "- load the model back from storage, to use in other notebooks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Install dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "885e8968-cc18-48db-af92-1b034c6ea3af",
+     "showTitle": true,
+     "title": "Install dependencies"
+    },
+    "jupyter": {
+     "outputs_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%pip install mlflow torch\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Imports + data preparation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "092ae65d-cebd-4778-8db5-400097966a3e",
+     "showTitle": true,
+     "title": "Imports + data preparation"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import torch\n",
```

Comment:
Let's split the import and data preprocessing code. Also we need to briefly explain what this dataset is, and what kind of proprocessing we are doing.